pkgs/development/compilers/llvm/16/libcxxabi/default.nix

```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [ cmake ninja python3 ];
-  buildInputs = lib.optional (!stdenv.isDarwin && !stdenv.hostPlatform.isWasm) libunwind;
+  propagatedBuildInputs = lib.optional (!stdenv.isDarwin && !stdenv.hostPlatform.isWasm) libunwind;
```

Comment:
Moved to propagatedBuildInputs, otherwise I get link errors for unwind symbols downstream (e.g. building libcxx).

Reply:
`unwind` is not mentioned in the header file, so it does not appear to be necessary to propagate it. Other builds should explicitly depend on it.

Reply:
Perhaps some changes are needed elsewhere if the unwind is perhaps not bundled any more somewhere?

Reply:
Maybe worth looking in to 
https://github.com/NixOS/nixpkgs/blob/82b98eb23fbdc46226bd3cc0b9632a31361d2090/pkgs/development/compilers/llvm/16/default.nix#L157-L161
https://github.com/NixOS/nixpkgs/blob/82b98eb23fbdc46226bd3cc0b9632a31361d2090/pkgs/development/compilers/llvm/16/default.nix#L242-L263

Im not familiar with the LLVM ecosystem

Reply:
Thanks for the review. I think you're right; I think that this is handled for most things in clangUseLLVM, and where it's missing is in a piece that feeds into the libcxx build, possibly `clangNoLibcxx`? I intend to look into this soon.

Reply:
Removed it and included it directly in libcxx.